.symfony.bundle.yaml

```diff
@@ -8,5 +8,5 @@ maintained_branches:
     - "3.5.x"
```

Comment:
Sorry, I caused this issue. The right thing to do is to remove `3.5.x` from `maintained_branches` I think.

```suggestion
```